File: ios/CustomMethods.m
//
//  CustomMethods.m
//  vectordesign
//
//  Created by Luan on 17/01/2024.
//

#import <Foundation/Foundation.h>
#import "React/RCTBridgeModule.h"

@interface RCT_EXTERN_MODULE(CustomMethods, NSObject)

RCT_EXTERN_METHOD(MyMethod : (NSString *) param)

@end

File: ios/CustomMethods.swift
//
//  CustomMethods.swift
//  vectordesign
//
//  Created by Luan on 17/01/2024.
//

import Foundation

@objc(CustomMethods) class CustomMethods: NSObject {
  @objc static func requireMainQueueSetup() -> Bool {return true}
  
  @objc public func MyMethod (_ param: String) {
    print(param)
    
    RNEventEmitter.emitter.sendEvent(withName: "onReady", body: [param])
  }
}
